packaging/windows/OpenRA.nsi

```diff
@@ -227,7 +229,19 @@ FunctionEnd
 !insertmacro Clean "un."
 
 Section "Uninstall"
+	${nsProcess::FindProcess} "OpenRa.Game.exe" $R0
+	IntCmp $R0 0 gameRunning
+	${nsProcess::FindProcess} "OpenRa.exe" $R0
+	IntCmp $R0 0 gameRunning
+	${nsProcess::FindProcess} "OpenRa.Editor.exe" $R0

```

Comment:
Well, there has even been a bug already filed for it https://github.com/OpenRA/OpenRA/issues/7142 so I guess that is a problem (maybe for dedicated servers running silently in the background). Just fix the "OpenRa..." filenames.